src/components/personkort/PersonkortChangeEnhet.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from "react";
+import { BodyLong, Button, Heading, Modal } from "@navikt/ds-react";
+import { BehandlendeEnhet } from "@/data/behandlendeenhet/types/BehandlendeEnhet";
+import styled from "styled-components";
+import { useChangeEnhet } from "@/components/personkort/useChangeEnhet";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import ModalWrapper from "nav-frontend-modal";
+
+const texts = {
+  endre: "Endre til",
+  innholdModal:
+    "Hvis du ikke har tilgang til NAV utland, vil du miste tilgangen til denne personen når enheten endres.\n" +
+    "Veiledere med tilgang til NAV utland kan senere flytte personen tilbake til geografisk enhet.",
+};
+
+const NAV_UTLAND = "0393";
+
+const ButtonGroup = styled.div`
+  > * {
+    margin-right: 1em;
+  }
+`;
+
+interface PersonkortChangeEnhetProps {
+  behandlendeEnhet: BehandlendeEnhet;
+}
+
+const PersonkortChangeEnhet = ({
+  behandlendeEnhet,
+}: PersonkortChangeEnhetProps) => {
+  const [open, setOpen] = useState(false);
+  const fnr = useValgtPersonident();
+  const changeEnhet = useChangeEnhet(fnr);
+
+  const isCurrentlyNavUtland = behandlendeEnhet.enhetId === NAV_UTLAND;
+  const heading = `${texts.endre} ${
+    isCurrentlyNavUtland ? "geografisk enhet" : "NAV utland"
+  }`;
+
+  const updateEnhet = () => {
+    changeEnhet.mutate({
+      personident: fnr,
+      isNavUtland: !isCurrentlyNavUtland,
+    });
+    setOpen(false);
+  };
+
+  useEffect(() => {
+    if (Modal.setAppElement) {
+      Modal.setAppElement("body");
+    }
+  }, []);
```

Comment:
Dette er for å unngå følgende warning i consollen:
https://stackoverflow.com/questions/48269381/warning-react-modal-app-element-is-not-defined-please-use-modal-setappeleme

Har testet med min begrensede kunnskap i VoiceOver, virker som om den leser riktig (leser hvertfall ikke "resten av siden" bak modalen)

Reply:
Fjernet grunnet test-feil